components/desktop/pulseaudio/files/pulse_exec_attr

```diff
@@ -0,0 +1 @@
+Basic Solaris User:solaris:cmd:RO::/usr/bin/pulseaudio:privs=proc_priocntl,file_dac_read
```

Comment:
On Linux it uses capabilities to access what it should and raise the priority within the range assigned in the system configuration but beyond its original priority.
On Solaris it is conservative, without the privilege the process can only lower its priority.